examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -21334,6 +21633,22 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
```

Comment:
What do these changes mean?

I am generally trying to have .matter files be a human readable version of changes to zap files (since those are not human readable jsons ... this is for example a delta at line 21K+). However these changes do NOT show up in a matter file.

A followup is ok because matter parsing and generation is more complex and could be done in parallel, however what do these mean and how should they be tested?

Reply:
@andy31415 I'll follow up on this. But in a high level I believe that these Global attributes are intentionally excluded from ZAP and MATTER files because they are assumed to be present by default.